ConsoleApp1/ServerRoom.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleApp1
+{
+   
+        public class Showall
+        {
+            Server newServer = new Server();
+            public void show()
+            {
+                for (int x = 0; x < newServer.checker; x++)
+                {
+                    Console.WriteLine(newServer.name[x]);
+                    Console.WriteLine(newServer.middleinitial[x]);
+                    Console.WriteLine(newServer.yearlevel[x]);
+                    Console.WriteLine(newServer.course[x]);
+                }
+            }
+        }
+
+        public class Account
+        {
+            Student newStudent = new Student();
+            public void newaccount()
+            {
+                Console.WriteLine("\n\nname:");
+                string name = Console.ReadLine();
+                Console.WriteLine("middle initial:");
+                char minitial = Convert.ToChar(Console.ReadLine());
+                Console.WriteLine("year level:");
+                int yl = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("course:");
+                string course = Console.ReadLine();
+
+                newServer.setname(name);
+                newServer.setmiddleinitial(minitial);
+                newServer.setyearlevel(yl);
+                newServer.setcourse(course);
+                newServer.checkerplusone();
```

Comment:
change this to newStudent.Name etc...hahaha

Reply:
newServer.setname = name;
                newServer.setmiddleinitial = minitial;
                newServer.setyearlevel = yl;
                newServer.setcourse = course;

Reply:
awhhh lahi lagi

Reply:
abi nako kailangan pag brackets

Reply:
setName(name) -> method ni siya hehehe
kanang NewStudent.Name = name; -> setters and getters ni siya, imagin lang nga ang table mao na si student unya ang naa sa table kay kanang name, course, level etc...

Reply:
ahhhwh orayt Noted!